### ISSUE TITLE
test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_VRFY Failed

### ISSUE THREAD

**Issue by RussTreadon-NOAA** [NOAA-EMC/GDASApp#1153]
Build GDASApp `develop` at 5071c52 inside g-w.  Ran `test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_VRFY`.  The test fails with the traceback
```
2024-06-07 11:59:36:INFO:Now processing: insitu_surface_trkob_salinity
2024-06-07 11:59:36:INFO:Writing replaced template to preevayamls/eva_insitu_surface_trkob_salinity_2018041512.yaml
2024-06-07 11:59:36:INFO:Loading input YAML from preevayamls/eva_insitu_profile_tesac_2018041512.yaml
Traceback (most recent call last):
  File "/scratch1/NCEPDEV/da/Russ.Treadon/git/global-workflow/pr2641/sorc/gdas.cd/scripts/exgdas_global_marine_analysis_vrfy.py", line 187, in <module>
    marine_eva_post.marine_eva_post(infile, 'evayamls', diagdir)
  File "/scratch1/NCEPDEV/da/Russ.Treadon/git/global-workflow/pr2641/sorc/gdas.cd/ush/eva/marine_eva_post.py", line 39, in marine_eva_post
    layer['vmin'] = vminmax[variable]['vmin']
KeyError: 'waterTemperature'
```

**Comment by RussTreadon-NOAA** [comment]
@apchoiCMD , will issue #1145 fix the failing `test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_VRFY`?

**Comment by apchoiCMD** [comment]
> @apchoiCMD , will issue #1145 fix the failing `test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_VRFY`?

@RussTreadon-NOAA I think I will do it, but this issue is unknown until you let me know. Let me tag up with @guillaumevernieres -

**Comment by JohnSteffen-NOAA** [comment]
I was able to replicate this error and find a fix. The variable definitions with max/min values for "waterTemperature" and "salinity" are not included in the /ush/eva/marine_eva_post.py script. I have added the following script changes to that file and it runs successfully. I will include this in a future PR to fix bugs in the marine vrfy task:

Modify /ush/eva/marine_eva_post.py
```
-           'absoluteDynamicTopography': {'vmin': -0.2, 'vmax': 0.2}}
+           'absoluteDynamicTopography': {'vmin': -0.2, 'vmax': 0.2},
+           'waterTemperature': {'vmin': -2.0, 'vmax': 2.0},
+           'salinity': {'vmin': -0.2, 'vmax': 0.2}}
```

**Comment by guillaumevernieres** [comment]
this will be addressed in #1314
